Song.Tests/SongWriterTests.cs

```diff
@@ -0,0 +1,132 @@
+using System.Text;
+
+namespace Song.Tests
+{
+    public class SongWriterTests
+    {
+        [Fact]
+        public void Sing_SingleAnimal()
```

Comment:
Tests aren't self-documenting, don't explain how it works.

Reply:
Renamed the tests.